OnlinerTask/OnlinerTask.BLL/Services/ElasticSearch/ProductLogger/IProductLogger.cs

```diff
@@ -0,0 +1,12 @@
+﻿using System.Threading.Tasks;
+using OnlinerTask.Data.SearchModels;
+
+namespace OnlinerTask.BLL.Services.ElasticSearch.ProductLogger
+{
+    public interface IProductLogger
+    {
+        Task LogAdding(int productId, ProductModel productModel);
+
+        Task RemoveLog(int productId);
```

Comment:
So lets add another type of logs - product deletion. It is also useful for business to have this kind of info.